conf/messages.de

```diff
@@ -403,3 +407,4 @@ footer.logo.google.alt = Google-Logo
 footer.logo.sloan.alt = Logo der Alfred P. Sloan Foundation
 footer.logo.pactrans.alt = Logo des Pacific Northwest Transportation Consortium (PacTrans).
 footer.logo.create.alt = Logo des Zentrums für Forschung und Bildung zu barrierefreien Technologien und Erfahrungen (CREATE).
+
```

Comment:
It doesn't actually matter (because there are no issues with having two newlines at the ends of files), but you actually went from having no blank line at the ends of files to having two! Just wanted to point it out in case there is some issue with a setting in your IDE or something that you might want to look into? Again, not a big deal and I wouldn't bother to send you back to fix it lol

Reply:
Must have been a mistake! Checked the .de file and yes, there were 2 newlines :)